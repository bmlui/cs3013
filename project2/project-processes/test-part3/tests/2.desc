Test history, more than 10 commands 
