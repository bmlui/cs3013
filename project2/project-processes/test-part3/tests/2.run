./wshell < tests/2.in
