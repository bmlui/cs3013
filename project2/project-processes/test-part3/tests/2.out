test-part3$ echo 1
1
test-part3$ echo 2
2
test-part3$ echo 3
3
test-part3$ echo 4
4
test-part3$ echo 5
5
test-part3$ echo 6
6
test-part3$ echo 7
7
test-part3$ echo 8
8
test-part3$ echo 9
9
test-part3$ echo 10
10
test-part3$ echo 11
11
test-part3$ history
echo 3
echo 4
echo 5
echo 6
echo 7
echo 8
echo 9
echo 10
echo 11
history
test-part3$ exit
